src/tasks/mt.py

```diff
@@ -44,6 +44,7 @@ def __init__(self, path, max_seq_len, max_targ_v_size, name, **kw):
         self.files_by_split = {
             split: os.path.join(path, "%s.txt" % split) for split in ["train", "val", "test"]
         }
+        log.warning("Machine translation is not fully supported in Jiant.")
```

Comment:
=> 'jiant'
And maybe add some spacing or exclamation points. This could be easy to miss.